scripts/test.sh

```diff
@@ -1,8 +1,18 @@
 #! /bin/bash
 set -e
+
+# Virtual token is only needed for tests. As utilities' interface may vary with version, don't create token in different environments
```

Comment:
Duplicated code(with coverage.sh) and finctionality with my PR #351

Reply:
Almost. It turned out that in order for TLS to work properly, keys should be imported with 'softhsm2-util --import'.